pages/common/tlmgr-path.md

```diff
@@ -0,0 +1,13 @@
+# tlmgr path
+
+> Add or remove symlinks for TeX Live executables, man pages and info pages.
```

Comment:
`executables, man pages and info pages` sounds kinda weird to me, but I'm not sure, how to improve on it either

Reply:
the only thing I would change about it is making man - manual and info to information